Sources/CollectionViewDriver.swift

```diff
@@ -151,15 +158,21 @@ public final class CollectionViewDriver: NSObject {
         self._cachedRegistrations.formUnion(newRegistrations)
     }
 
-    private func _didUpdateModel(from old: CollectionViewModel, to new: CollectionViewModel) {
+    private func _updateViewModel(
+        from old: CollectionViewModel,
+        to new: CollectionViewModel,
+        animated: Bool,
+        completion: DidUpdate?
+    ) {
+        self.viewModel = new
```

Comment:
@clevin95 Curious about your thoughts on this piece here.

`viewModel` is `@Published`, so you can `.sink { }` on this property.  Setting the new value here updates the property immediately. Alternatively, we _could_ delay this until after animations complete. I'm curious what behavior you would expect?

My thinking at the moment is:

This is the most "accurate". This is when the value is actually updated. If consumers want to be notified of updates _after_ animations complete, then they should use `completion`.

Reply:
Seems reasonable to me. It would be confusing if the view model weren't updated immediatly after calling `update`.

Makes me wonder if we should be exposing the viewModel at all thought.

Reply:
> Makes me wonder if we should be exposing the viewModel at all though.

That's fair. It is immutable/read-only, so there's no risk. I'll leave it for now and revisit later if needed.